Dockerfile

```
@@ -3,7 +3,7 @@ FROM debian:buster
 ARG AWSCLI_VERSION=1.25.38
 ARG AZURECLI_VERSION=2.17.1
 ARG CLOUD_SDK_VERSION=336.0.0
-ARG HELM_VERSION=v3.8.2
+ARG HELM_VERSION=v3.12.0
```

Seems that the latest is 3.12.1
```suggestion
ARG HELM_VERSION=v3.12.1
```

---

Hello, v3.12.1 was not not announced as stable on helm web site. But I can see it is available on the repo